File: java/encapsulamiento/Ejemplo01/mx/uaemex/data/Persona.java
package mx.uaemex.data;

public class Persona
{
    private String nombre;
    private int edad;
    private float estatura;
    
    public void setEstatura(float estatura) {
        this.estatura = estatura;
    }
    
    public float getEstatura() {
        return estatura;
    }
    
    public void setEdad(int edad) {
        this.edad = edad;
    }
    
    public int getEdad() {
        return edad;
    }
    
    public void setNombre(String nombre) {
        //Verificar que nombre contenga al menos 2 caracteres
        //En caso contrario, escribir en consola:
        //"Error en el nombre, debe tener al menos 2 letras"
        if ( nombre.length() < 2) {
            System.out.println("Error en el nombre, debe tener al menos 2 letras");
            this.nombre = "-Indefinido-";
        } else {
            this.nombre = nombre;
        }
    }
    
    public String getNombre(){
        return nombre;
    }
    
    public void verInfo() {
        System.out.println("Datos:\n" + 
            "Nombre: " + nombre + "\n" + 
            "Edad: " + edad + "\n" + 
            "Estatura: " + estatura);
    }
}

File: python/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.1 Herencia/NominaV3/Persona.py
class Persona:
    def __init__(self, nombre, edad, estatura):
        self.nombre = nombre
        self.edad = edad
        self.estatura = estatura

    def getNombre(self):
        return self.nombre

    def getEdad(self):
        return self.edad

    def getEstatura(self):
        return self.estatura

    def __str__(self):
        return f"nombre='{self.nombre}', edad='{self.edad}', estatura='{self.estatura}'"
